models/remoteworkitem.go

```diff
@@ -0,0 +1,67 @@
+package models
+
+import (
+	"encoding/json"
+)
+
+const (
+	SystemRemoteIssueId = "system.remote_issue_id"
+	SystemTitle         = "system.title"
+	SystemDescription   = "system.description"
+	SystemStatus        = "system.status"
+	ProviderGithub      = "github"
```

Comment:
Thanks.

Hi @baijum FYI.
This `const` declaration holds the key of the `trackerTypes` map in models/tracker_respository.go